packages/cli-repl/src/arg-mapper.spec.ts

```diff
@@ -315,6 +436,11 @@ describe('arg-mapper.mapCliToDriver', () => {
           strict: true,
           deprecationErrors: true,
           version: '1'
+        },
+        driverInfo: {
+          name: 'mongosh',
+          platform: 'darwin',
+          version: '0.0.0-dev.0'
```

Comment:
Won’t this assertion fail when we try to do a release, where the version will be set to something different?

Reply:
Yes, it will, but I thought the idea is to not change this value. I took the idea from what Maurizio added to vscode: https://github.com/mongodb-js/vscode/pull/315/files#diff-a884cbf73a5998822710b6ce02a6bdb680fb24427451911940539004b83e3dbeR515

What would be a better approach here?

Reply:
> I thought the idea is to not change this value.

Well, CI changes that value when we release and before we run the tests, so that’s not really going to work

> What would be a better approach here?

You can also use `require('../package.json').version` here. I know if feels a bit silly to use the same code in the test and in the source code, but I don’t really have better ideas :woman_shrugging: It’s also not entirely without precedent: https://github.com/mongodb-js/mongosh/blob/e6e75b455a65fad7b78c42571d170bc5cc1891ef/packages/cli-repl/test/e2e.spec.ts#L232

Reply:
I didn't know that ci changes it, yeah in this case it won't work. I will require package.json then.